src/components/Search/SelectedRevisionItem.tsx

```diff
@@ -82,14 +81,33 @@ function SelectedRevisionItem({
   }, [selectedRevisions]);
 
   const handleClose = () => {
-    if (view == 'search') {
-      handleRemoveRevision(item);
+    if (view == searchView || !editBtnVisible) {
+      removeCheckedRevision(item);
     } else {
       deleteSelectedRevisions(item);
     }
     return;
   };
 
+  const renderCloseIcon = () => {
+    const isSearchViewOrEditBtnVisible = view === searchView || !editBtnVisible;
+    const isBaseAndEditBtnVisible = searchType === 'base' || !editBtnVisible;
+
+    const iconClassName = isSearchViewOrEditBtnVisible
+      ? 'icon icon-close'
+      : isBaseAndEditBtnVisible
+      ? 'icon icon-close-base'
+      : 'icon icon-close results';
```

Comment:
I don't see where the CSS for `results` is, am I missing something?

Reply:
Yeah, I simplified this. I don't know what I was thinking here lol. This is why I appreciate PR reviews lol.